.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing Guidelines
+
+* You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/).  Each active repo has its own stream to direct questions to (for example #powerup or #portal).  STEM-diverse-tv stream is [#STEM-diverse-tv](https://anitab-org.zulipchat.com/#narrow/stream/225705-STEM-diverse-tv).
```

Comment:
powerup is no longer supported by AnitaB.org

Reply:
@annabauza okayy, I've replaced that with mentorship-system :)

Reply:
👍